src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/debugger/windbg/WindbgDebugger.java

```diff
@@ -51,9 +51,8 @@ public long         readCInteger(long address, long numBytes, boolean isUnsigned
   public WindbgOopHandle readCompOopHandle(long address) throws DebuggerException;
 
   // The returned array of register contents is guaranteed to be in
-  // the same order as in the DbxDebugger for Solaris/x86 or amd64; that is,
-  // the indices match those in debugger/x86/X86ThreadContext.java or
-  // debugger/amd64/AMD64ThreadContext.java.
+  // the same order as in the DbxDebugger for Solaris/amd64; that is,
+  // the indices match those in debugger/amd64/AMD64ThreadContext.java.
```

Comment:
I would just delete this comment block as we no longer support Solaris and the reference to dbx is obsolete and we have no idea if we are maintaining order with any other debuggers.